inc/JpegMeta.php

```diff
@@ -1420,61 +1420,67 @@ function _parseMarkerJFIF() {
             $this->_readJPEG();
         }
 
-        if ($this->_markers == null) {
+        if ($this->_markers == null || $this->_isMarkerDisabled(('jfif'))) {
             return false;
         }
 
-        $data = null;
-        $count = count($this->_markers);
-        for ($i = 0; $i < $count; $i++) {
-            if ($this->_markers[$i]['marker'] == 0xE0) {
-                $signature = $this->_getFixedString($this->_markers[$i]['data'], 0, 4);
-                if ($signature == 'JFIF') {
-                    $data =& $this->_markers[$i]['data'];
-                    break;
+        try {
+            $data = null;
+            $count = count($this->_markers);
+            for ($i = 0; $i < $count; $i++) {
+                if ($this->_markers[$i]['marker'] == 0xE0) {
+                    $signature = $this->_getFixedString($this->_markers[$i]['data'], 0, 4);
+                    if ($signature == 'JFIF') {
+                        $data =& $this->_markers[$i]['data'];
+                        break;
+                    }
                 }
             }
-        }
 
-        if ($data == null) {
-            $this->_info['jfif'] = false;
-            return false;
-        }
+            if ($data == null) {
+                $this->_info['jfif'] = false;
+                return false;
+            }
 
-        $this->_info['jfif'] = array();
+            $this->_info['jfif'] = array();
 
-        $vmaj = $this->_getByte($data, 5);
-        $vmin = $this->_getByte($data, 6);
+            $vmaj = $this->_getByte($data, 5);
+            $vmin = $this->_getByte($data, 6);
 
-        $this->_info['jfif']['Version'] = sprintf('%d.%02d', $vmaj, $vmin);
+            $this->_info['jfif']['Version'] = sprintf('%d.%02d', $vmaj, $vmin);
```

Comment:
There's mixed space/tab formatting here. Please adjust to 4 spaces.

Reply:
I'm sorry, but I've checked multiple times with multiple tools, and there's not a single tab in this file. Every indent should be 4 spaces wide.

Reply:
You're right. I misinterpreted the diff. my bad.